accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -150,115 +152,136 @@ class OptionsApp extends React.Component<{}, OptionsAppState> {
-                light={false}
-                titleText=""
-                type="default"
-                selectedItem={selected_ruleset}
-                onChange={this.handleRulesetSelect}
-              />
+                <p>
+                  Choose to always use the latest version of the rule sets, use
+                  the version from a specific date, or try a preview of future
+                  rule sets. By default the latest rule set version is selected.
+                </p>
 
-      {selected_ruleset.description? (<p className="op_helper-text">{selected_ruleset.description}</p>):"" }
-              {show_notif ? (
-                <div className="notification">
-                  <InlineNotification
-                    role="alert"
-                    kind="success"
-                    lowContrast={true}
-                    title="Success"
-                    subtitle=" Your changes have been saved"
-                    className=""
-                    iconDescription="close notification"
-                    onCloseButtonClick={() => {
-                      this.setState({ show_notif: false });
-                    }}
-                  />
-                </div>
-              ) : (
-                ""
-              )}
-              <div className="buttonRow">
-                <Button
+                <Dropdown
+                  ariaLabel="Deployment selection dropdown"
                   disabled={false}
-                  kind="tertiary"
-                  onClick={this.handlReset}
-                  renderIcon={Restart16}
-                  size="default"
-                  tabIndex={0}
-                  type="button"
-                >
-                  Reset
-                </Button>
-                <Button
+                  helperText="Rule set deployment"
+                  id="archivedRuleset"
+                  items={archives}
+                  itemToString={(item: any) => (item ? item["name"] : "")}
+                  label="Rule set deployment"
+                  light={false}
+                  titleText=""
+                  type="default"
+                  selectedItem={selected_archive}
+                  onChange={this.handleArchiveSelect}
+                />
+                <p className="op_helper-text">
+                  For details on rule set changes between deployments, see{" "}
+                  <a
+                    href="https://github.com/IBMa/equal-access/releases"
+                    target="_blank"
+                    rel="noopener noreferred"
+                  >
+                    Release notes
+                  </a>
+                  .
+                </p>
+                <h2>Supported rule sets</h2>
+                <p>
+                  Choose which rule set to use. This will affect the issues
+                  detected.
+                </p>
+
+                <Dropdown
+                  ariaLabel="ruleset selection dropdown"
```

Comment:
Why is there both a label and an ariaLabel? Also, you should not put 'dropdown' in the label, much as you wouldn't add 'dropdown' to the visible label. Widget type is conveyed to the screen reader through the widget role.

Reply:
For Type script, it looks label and ariaLabel are both required by Dropdown component ,  removing either one of them will cause compilation issue, even the Carbon's doc saying ariaLabel is optional,
https://react.carbondesignsystem.com/?path=/story/dropdown--default

I know the voice over speaks both arialLabel and label. it is kind of redundant. To work around, I will keep the arialLabel, but assign an empty string to it like this: ariaLabel="". Hope it is OK.

Reply:
I took a look at the storybook. Looks like they set an aria-label on the div surrounding the button with no role. Looks like bad implementation. Try `ariaLabel={null}`.

Reply:
tried ariaLabel={null}, it is not taken: 
![image](https://user-images.githubusercontent.com/14806866/86388975-43936380-bc5b-11ea-8431-a665f8657cc9.png)

So I changed to ariaLabel={undefined}, and it looks fine.

Reply:
Did anyone have a chance to check it with a screen reader? The DOM looked okay to me, but would be nice to confirm that there isn't something weird going on.

Reply:
I did Voiceover checking ( have not got time to install JAWS), the label read correctly.